cas-server-documentation/installation/Maven-Overlay-Installation.md

```diff
@@ -168,5 +168,53 @@ in the Maven overlay must include a reference to the Maven Java compiler so clas
 
 ```
 
+## Dependency Management
+
+Each release of CAS provides a curated list of dependencies it supports. In practice, you do not need to provide a version for any of 
+these dependencies in your build configuration as the CAS distribution is managing that for you. When you upgrade CAS itself, 
+these dependencies will be upgraded as well in a consistent way.
+
+The curated list contains a refined list of third party libraries. The list is available as a standard Bills of Materials (BOM).
+
+To configure your project to inherit from the BOM, simply set the parent:
+
+```xml
+<!-- Inherit defaults from Spring Boot -->
+<parent>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-bom</artifactId>
+    <version>${cas.version}</version>
+</parent>
+```
+
+Not everyone likes inheriting from the BOM. 
+You may have your own corporate standard parent that you need to use, 
+or you may just prefer to explicitly declare all your Maven configuration.
+
+If you don’t want to use the spring-boot-starter-parent, you can still 
+keep the benefit of the dependency management (but not the plugin management) 
+by using a `scope=import` dependency:
+
+```xml
+<dependencyManagement>
+     <dependencies>
+     
+        <!-- Override a dependency by including it BEFORE the BOM -->
+        <dependency>
+            <groupId>org.group</groupId>
+            <artifactId>artifact-name</artifactId>
+            <version>X.Y.Z</version>
+        </dependency>
+             
+        <dependency>
+            <groupId>org.apereo.cas</groupId>
+            <artifactId>cas-server-support-bom</artifactId>
+            <version>${cas.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+```
 
```

Comment:
And also it'd be nice to generate a list of all the dependencies and versions provided by the BOM... somehow?